src/core/scene/Scene.js

```diff
@@ -1161,6 +1161,7 @@ var Scene = Class.$extend(
             var entityId = ds.readVLE();
             var entity = this.entityById(entityId);
             var placeable = entity ? entity.placeable : null;
+            var rigidBody = entity ? entity.getComponent("EC_RigidBody") : null;
```

Comment:
We have shorthands for both all lower and the original comp name `entity.rigidBody` `entity.rigidbody`. A bit easier on the eye and also faster as getComponent will iterate stuff. If you have multiple components with certain type, then  getters need to be used.

**Note:** `ent.name` is an exception, it wont return EC_Name but the property or EC_Name.name if available.